services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/pathverification/PathVerificationService.java

```diff
@@ -366,22 +369,23 @@ private LLDPTLV makeIdLldptvPacket(byte[] data, byte type) {
         return new LLDPTLV().setType(type).setLength((short) data.length).setValue(data);
     }
 
-    private VerificationPacket deserialize(Ethernet eth) throws Exception {
+    VerificationPacket deserialize(Ethernet eth) {
```

Comment:
minor: Why did you remove private access modifier?

Reply:
for testing. Will add annotation